### ISSUE TITLE
cube intrinsics

### ISSUE THREAD

**Issue by Golbstein** [thucz/splatter360#7]
I've noticed cube instrinsics are

```
[[0.4980, 0.0000, 0.5000], 
[0.0000, 0.4980, 0.5000],
[0.0000, 0.0000, 1.0000]],
```

but from what I see each cube is at FOV=90degrees so where this 0.498 value comes from? It should be 0.5 no?

**Comment by thucz** [comment]
You are right. Our dataset generation code has a bug in the [preprocessing code](https://github.com/thucz/splatter360/blob/a6208857905bb47f7183c67818927c40b370ce4c/preprocess/convert_cubemaps_mp.py#L144). Now we fix it. Actually `fx=fy=cx=cy=0.5` in HM3D and Replica.
